### PR TITLE
Collection of minor improvements to the code

### PR DESCRIPTION
**Pull request by Geod24**
Working on fixing then numerous bugs we found in production, ended up with a bunch of simple commits.

### CODE REVIEW

File: src/modules/data/Height.ts [hunk 1]
```diff
@@ -30,12 +30,11 @@ export class Height
      * Construct
      * @param value - The block height
      */
-    constructor (value?: UInt64)
+    constructor (value: bigint);
+    constructor (value: UInt64);
+    constructor (value: any)
```
Comment (MichaelKim20) [inline src/modules/data/Height.ts]: How about this?
```suggestion
    constructor (value: bigint | string | UInt64)
    {
        if (typeof value == "bigint")
        {
            this.value = UInt64.fromString(value.toString());
        }
        else if (typeof value === "string")
        {
            this.value = UInt64.fromString(value);
        }
        else
        {
            this.value = new UInt64(value);
        }
    }
```

Comment (Geod24) [reply src/modules/data/Height.ts]: Good suggestion, I committed a slightly modified version.

**Comment by Geod24** [comment]
Should be fixed now.

**Comment by Geod24** [comment]
Hum, shall we drop support for 10.x and 12.x ?

**Review by MichaelKim20 — APPROVED** [review]

**Pull request merged**